tests/migration/online_migration/yast2_migration.pm

```diff
@@ -189,25 +190,33 @@ sub run {
                 }
             }
             if (match_has_tag 'yast2-migration-inconsistency') {
-                wait_screen_change { send_key 'alt-c' };    # continue
+                wait_screen_change { send_key 'alt-c' };
```

Comment:
I think reserve the comments will be helpful for code understanding.

Reply:
changed. thanks